scribbles/apps/authentication/schema.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth import get_user_model
+import graphene
+from graphene_django import DjangoObjectType
+
+
+class UserType(DjangoObjectType):
+    class Meta:
+        model = get_user_model()
+
+
+# testing authentication
+class Query(graphene.AbstractType):
+    me = graphene.Field(UserType)
+    users = graphene.List(UserType)
+
+    def resolve_users(self, info):
+        return get_user_model().objects.all()
+
+    def resolve_me(self, info):
+        user = info.context.user
+        if user.is_anonymous:
+            raise Exception('Not logged in!')
+
+        return user
+
+
+class CreateUser(graphene.Mutation):
+    user = graphene.Field(UserType)
+
+    class Arguments:
+
+        username = graphene.String(required=True)
+        email = graphene.String(required=True)
+        password = graphene.String()
+
+    def mutate(self, info, username, email, password):
+        user = get_user_model()(username=username, email=email, password=password)
+        user.set_password(password)
+        user.save()
+        return CreateUser(user=user)
+
+
+class Mutation(graphene.ObjectType):
```

Comment:
Create a folder called schema, and then have two folders inside it, one for mutations and the other for queries. We'll adding all mutations and queries related to users in these two folders  e.g create_user_mutation.py (inside mutations folder) and users_query.py (inside queries folder)